Configuration/JavaScriptModules.php

```diff
@@ -7,6 +7,7 @@
         'backend.contextmenu',
     ],
     'imports' => [
+        'jquery-ui/' => 'EXT:core/Resources/Public/JavaScript/Contrib/jquery-ui/',
```

Comment:
Still not sure if this is required as import, but did not checked that yet (and not a js pro to have that in the brain). Sadly, not time was given to review and analyse this, even if it would have been ruled out to be valid and the correct approach.

Reply:
drag-drop.js is using jquery-ui and from core-imports the `Contrib`-folder is exluded in the wildcard-import.